.github/workflows/nightly-build.yml

```diff
@@ -28,6 +27,17 @@ jobs:
           fetch-depth: 0
           submodules: 'true'
 
+      - name: Cache build output
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/Library/Caches/go-build
```

Comment:
This cache path looks incorrect for ubuntu.

Reply:
Fixed it in the newest commit. Nice catch on that though.